samples/instanceadmin/test_instanceadmin.py

```diff
@@ -117,22 +123,29 @@ def test_delete_instance(capsys, dispose_of):
 
 
 def test_add_and_delete_cluster(capsys, dispose_of):
-    dispose_of(INSTANCE)
+    from concurrent.futures import TimeoutError
 
-    # This won't work, because the instance isn't created yet
-    instanceadmin.add_cluster(PROJECT, INSTANCE, CLUSTER2)
-    out = capsys.readouterr().out
-    assert f"Instance {INSTANCE} does not exist" in out
+    @backoff.on_exception(backoff.expo, TimeoutError)
+    def _set_up_instance()
+        dispose_of(INSTANCE)
 
-    # Get the instance created
-    instanceadmin.run_instance_operations(PROJECT, INSTANCE, CLUSTER1)
-    capsys.readouterr()  # throw away output
+        # This won't work, because the instance isn't created yet
```

Comment:
This comment took a bit to understand. Are you saying we can't add a cluster because INSTANCE doesn't exist (and you ensured that in the line above)?

"get the instance created" also feels weird. Are we getting the instance or creating it? I had similar feelings about those above. I recognize those were carried forward.